demo/video/src/Video/DemoConference.tsx

```diff
@@ -0,0 +1,12 @@
+import { useState } from 'react';
+import { VideoConference } from '@signalwire-community/react';
+
+function DemoConference() {
+  return (
+    <div style={{ maxHeight: '100vh', aspectRatio: '16/9' }}>
+      <VideoConference token="vpt_78f91a752d4d9c685e47bd4a19fe72c1" />
```

Comment:
is this token from the `guides` repo?

Reply:
It's the same token we had in VideoConferences demo.